pytest/tests/loadtest/locust/README.md

```diff
@@ -98,8 +125,40 @@ Currently supported load types:
 | Sweat (normal load) | sweat.py | (`--sweat-wasm $WASM_PATH`) | Creates a single instance of the SWEAT contract. A mix of FT transfers and batch minting with batch sizes comparable to mainnet observations in summer 2023. |
 | Sweat (storage stress test) | sweat.py | `--tags=storage-stress-test` <br> (`--sweat-wasm $WASM_PATH`) | Creates a single instance of the SWEAT contract. Sends maximally large batches to mint more tokens, thereby touching many storage nodes per receipt. This load will take a while to initialize enough Sweat users on chain. |
 
-In the future, we might have multiple users per load type but for now there is a
-one-to-one mapping from users to load type (and tag).
+## Notes on Storage Stress Test
+
+The Sweat based storage stress test is special. While the other workloads send
+innocent traffic with many assumptions, this test pushes the storage accesses
+per chunk to the limit. As such, it is a bit more fragile.
+
+### Slow Start
+
+First, you will notice that for several minutes after start, it will only be
+registering new accounts to the Sweat contract on chain. You will see these
+requests show up as "Init FT Account" in the statistics tab of the Locust UI.
+
+To make sure you are not waiting forever, you want to have enough locust users
+per worker instance. For example, 100 users in one worker will be 100 times
+faster than 100 users distributed over 100 workers.
+
+Once enough accounts have been registered, large batches of users get new steps
+added, which translates to new tokens being minted for them. You will see them
+as `Sweat record batch (stress test)` requests.
+
+When you restart your workers, they reset their known registered user accounts.
+Hence on restart they add new accounts again. And you have to wait again. To
+avoid this, you can stop and restart tests from within the UI. This way, they
+will remember the account list and start the next test immediately, without long
+setup.
+
+
+### Master Key Requirements
+
+The `--funding-key` provided must always have enough balance to fund many users.
+But this is even more extreme for this load, as we are creating many accounts
+per worker.
 
-If the table above seems outdated, list available classes using `locust
---funding-key . --list` and send a PR to fix it, please.
+Also, the account name associated with this account should not bee too long.
+Keep it below 20 characters, and it should be fine. The reason is that we create
+sub accounts on this account, which become Sweat users. If the account names are
+too long, we hit the log output limit too quickly.
```

Comment:
I would move it to generic "Funding Key" section because I observed "too long account ID" error even in FT load. And it wasn't related to log output limit, it was just `invalid value: "be9qek_user83_run.mocknet-mainnet-74020501-logunov-new-mirror-rbvh.near", the Account ID is too long`

Reply:
Oh... Yeah then you hit the 64 bytes limit :) Good catch! I've now moved this to the general section, plus mentioned here that the limit is even tighter for Sweat stress testing.